bazel/src/test/data/googleapis/google/example/library/v1/BUILD.bazel.baseline

```diff
@@ -77,9 +77,9 @@ java_gapic_library(
     rest_numeric_enums = True,
     service_yaml = "library_example_v1.yaml",
     test_deps = [
-        ":library_java_grpc",
         "//google/cloud/location:location_java_grpc",
         "//google/iam/v1:iam_java_grpc",
+        ":library_java_grpc",
```

Comment:
nit: isn't the canonical order to have `:FOO` before `//BAR` in BUILD targets?

Reply:
Oh, I didn't know that, do you have a style guide link?

I didn't intentionally change the order, just kind of happened, didn't think it mattered.

Reply:
I'm not terribly worried about this, just a change I noticed. I thought the order of the leading punctuation did matter, but I can't find a reference to that in https://bazel.build/build/style-guide -- and all the examples have the targets `:` before `//`, but the target list also happens to be alphabetized. So maybe the principle is alphabetization-ignoring-leading-punctuation. Not clear from the guide. But if some official tool ordered things this way, that's probably it.

Summary: noop